modules/commandTxs.js

```diff
@@ -759,6 +759,21 @@ async function buy_sell(params, type) {
   };
 }
 
+/**
+ * Shows trading params
+ * @returns {Object} { msgNotify, msgSendBack, notifyType }
+ */
+function params() {
+  let output = `I am set to work with ${config.pair} pair on ${config.exchangeName}. Current trading settings:`;
+  output += '\n\n' + JSON.stringify(tradeParams, null, 3);
+
+  return {
+    msgNotify: '',
+    msgSendBack: `${output}`,
```

Comment:
`msgSendBack: output`

but I think it's better to rename output variable to `msgSendBack` to use ES6 features:

```js
const settings = JSON.stringify(tradeParams, null, 3); // use 2 or 4 spaces instead of 3
const msgSendBack = `I am set to work with ${config.pair} pair on ${config.exchangeName}. Current trading settings: \n\n${settings}`;

return {
  msgSendBack,
  msgNotify: '',
  // ...
};
```